client/src/components/OldHeader.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'React';
```

Comment:
Is there a reason why OldHeader _and_ Header are in this PR? Do we only need one file or both?